src/spikeinterface/core/sortinganalyzer.py

```diff
@@ -1615,7 +1612,8 @@ def load_data(self):
                 elif "object" in ext_data_.attrs:
                     ext_data = ext_data_[0]
                 else:
-                    ext_data = ext_data_
+                    # this load in memmory
+                    ext_data = np.array(ext_data_)
```

Comment:
@alejoe91 we need memory load otherwise the slicing will be buggy

Reply:
Ok, let's keep it as is at least for now